spec/services/billing_entities/update_service_spec.rb

```diff
@@ -208,7 +208,7 @@
 
           expect(result).not_to be_success
           expect(result.error).to be_a(BaseService::ValidationFailure)
-          expect(result.error.messages).to eq({eu_tax_management: ["org_must_be_in_eu"]})
+          expect(result.error.messages).to eq({eu_tax_management: ["billing_entity_must_be_in_eu"]})
```

Comment:
can we please add a test here that Taxes::AutoGenerate / ChangeEuTaxManagement service is called for eu billing_entity?

Reply:
it was already covered with tests, anyway I have took then chance to simplify the tests.